tests/playground.py

```diff
@@ -0,0 +1,86 @@
+# TLDR: `pip3 install -e .` from the root directory of the package
+# Then: `python3 ./tests/playground.py`
+#
+# development mode: "setuptools allows you to install a package without
+# copying any files to your interpreter directory
+# (e.g. the site-packages directory)."
+# From: https://setuptools.pypa.io/en/latest/userguide/quickstart.html#development-mode
+
+
+import resultwizard as wiz
+
+print("#############################")
+print("### Playground")
+print("#############################")
+print()
+
+
+wiz.config_init(print_auto=True, decimal_places=3)
+# wiz.config(sigfigs=2)
+# wiz.config(decimal_places=2)
+
+#############################
+# EXAMPLES
+#############################
+
+print("### RESULTS API")
+
+# wiz.res("", 42.0).print()
+# -> Error: "name must not be empty"
+
+wiz.res("a911", 1.0, r"\mm")
+# a: 1.0 \mm
+
+wiz.res("1 b", 1.0, 0.01, r"\per\mm\cubed").print()
+# b: (1.0 ± 0.01) \mm
+
+wiz.config(decimal_places=-1, sigfigs_fallback=3)
+
+wiz.res("c big", 1.0, (0.01, "systematic"), r"\mm").print()
+# c: (1.0 ± 0.01 systematic) \mm
+
+wiz.res(
+    "d", 1.0e10, [(0.01e10, "systematic"), (0.0294999999e10, "stat")], r"\mm\per\second\squared"
+).print()
+# d: (1.0 ± 0.01 systematic ± 0.02 stat) \mm
+
+wiz.res("e", "1.0", r"\mm").print()
+# e: 1.0 \mm
+
+wiz.res("f", "1.0e1", 25e-1).print()
+# f: 1.0
+
+wiz.res("g", 42).print()
+
+wiz.res("h", 42, 13)
+# TODO: 13 is misinterpreted as number of sigfigs
```

Comment:
We came to the conclusion that we want to allow integers only for the main value, not for uncertainties. This way, there's no ambiguity anymore and plum can correctly dispatch. Could you implement that @paul019?